src/database-assignment-schema.sql

```diff
@@ -0,0 +1,183 @@
+-- JOIN TABLES
+DROP TABLE IF EXISTS tv_compatible_ci_modules;
+DROP TABLE IF EXISTS tv_compatible_wall_brackets;
+-- SUBTYPES
+DROP TABLE IF EXISTS televisions;
+DROP TABLE IF EXISTS remote_controllers;
+DROP TABLE IF EXISTS ci_modules;
+DROP TABLE IF EXISTS wall_brackets;
+-- BASE PRODUCTS
+DROP TABLE IF EXISTS products;
+-- USER TABLES
+DROP TABLE IF EXISTS users;
+
+CREATE TABLE users
+(
+    id            SERIAL PRIMARY KEY,
+    username      VARCHAR(100) UNIQUE NOT NULL,
+    user_password VARCHAR(255)        NOT NULL, -- in realistic scenario I would store this as a secure hash, not plain text.
+    address       VARCHAR(255),
+    user_role     VARCHAR(20) CHECK (user_role IN ('admin', 'employee')),
+    pay_scale     INT                 NOT NULL CHECK (pay_scale >= 0),
```

Comment:
Hier lijkt per ongeluk veel witruimte tussen te zitten? Voor het runnen van het script maakt het niet uit, maar voor de leesbaarheid is het beter zonder